docs/utilities/idempotency.md

```diff
@@ -124,13 +155,15 @@ You can quickly start by initializing the `DynamoDBPersistenceLayer` class and u
     }
     ```
 
+After processing this request successfully, a second request containing the exact same payload above will now return the same response, ensuring our customer isn't charged twice.
+
```

Comment:
Should we add a text like "If you are not familiar with the concept of Idempotency, read the terminology" and add a link to #terminology?

I say this because sometimes I want to go straight to the example and I miss something of the concept that I should have read earlier.

Reply:
Done!